Cookbook/Technical-Documents/Optimizely.md

```diff
@@ -0,0 +1,89 @@
+Working with Optimizely
+=======================
+
+[Optimizely](https://www.optimizely.com) is a tool for full stack software experimentation. Its two main concepts are Experiments and Features. In an experiment users are divided into two or more segments, each segment is shown a different version of the user interface and we try to collect some performance data to decide which version works best. A feature is a flag which decides whether something is available for a specific user.
```

Comment:
Is it a "tool for full stack software experimentation" or more?
Your paragraph goes on to say it can be used for feature flags. What are they in this context, as I don't think they are what we think of as a feature flag generally within the app?